src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
```

Comment:
Maybe before the `moveJobToGroupModeTaskQueue ` and `moveJobToAddressModeTaskQueue `, we initialise `statusToAdmin` as `"Email queued:"` first. And then, in `moveJobToAddressModeTaskQueue`, we use `statusToAdmin += "Receipient: " ...`